React/Modules/MacOS/RCTAccessibilityManager.m

```diff
@@ -47,6 +55,24 @@ - (void)dealloc
                                         context:AccessibilityVoiceOverChangeContext];
 }
```

Comment:
Need to stop observing the notification here.

Reply:
I think this should successfully stop observing.